### DISCUSSION TITLE
Change font size in markdown table | VSCode

### DISCUSSION THREAD

**Discussion by jcmoral90** [marp-team/marp#217] [Q&A] [answered]
Hi everyone, I need to know how to reduce the font size of a table in markdown but make it fit the slide. The only thing that has worked for me is the following:

```
<style>
{
  font-size: 13px
}
</style>
```

But unfortunately, it changes the text size of all the slides. I would appreciate a solution.

![imagen](https://user-images.githubusercontent.com/64127850/142743277-97de085a-1079-4785-841c-0807f8e7fae5.png)

**Comment by yhatt** [comment, marked as answer]
```html
<style scoped>
table {
  font-size: 13px;
}
</style>
```

Marp recognizes [`<style scoped>`](https://marpit.marp.app/theme-css?id=scoped-style), to apply defined styles only to the current slide.

FYI the style declaration without [the selector](https://developer.mozilla.org/en-US/docs/Glossary/CSS_Selector) is actually invalid CSS, even if worked in Marp. You must set the target element of the style like above (`table` in this case).
